.github/workflows/sycl_precommit.yml

```diff
@@ -40,3 +40,9 @@ jobs:
       build_artifact_suffix: "default"
       build_cache_suffix: "default"
       lts_config: "hip_amdgpu;ocl_x64;ocl_gen9;l0_gen9"
+
+  windows_default:
+    name: Window
+    needs: lint
+    if: always() && (success() || contains(github.event.pull_request.labels.*.name, 'ignore-lint'))
```

Comment:
@alexbatashev, are we ready to enable it in pre-commit?
I have concerns about that.
Can we enable nightly and post-commit validation on Windows first and bring pre-commit later?
I'd like to make sure that these checks do not introduce false positives and we have enough resources to handle pre-commit requests.

Reply:
done